src/PageContainer.js

```diff
@@ -18,10 +19,24 @@ class PageContainer extends Component {
 
         this.state = {
             content: props.content,
-            donatingAgency: null
+            donatingAgency: null,
+            dialogOpen: false
```

Comment:
Don't need to change anything now, but I'm wondering if  `dialogOpen` field should actually end up being in the `Calendar` component?